### PR TITLE
Add README.pt.adoc Translate to portuguese: examples cdi basic TOMEE …

### PR DESCRIPTION
**Pull request by aasampaio3006**
…2516

### CODE REVIEW

File: examples/cdi-basic/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,169 @@
+= CDI @Inject
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+Para usar `@Inject`, a primeira coisa que você precisa de um arquivo `META-INF/beans.xml` arquivo no modulo
```
Comment (Daniel-Dos) [inline examples/cdi-basic/README_pt.adoc]: word "arquivo" two times

**Comment by Daniel-Dos** [comment]
small change :

-  change  "Add README.pt.adoc Translate to portuguese: examples cdi basic TOMEE " to "Translate to portuguese: examples cdi basic TOMEE-2516"

- insert small description with link of JIRA

**Comment by Daniel-Dos** [comment]
thanks , very good your first PR : )

File: examples/cdi-basic/README_pt.adoc (hunk 1 above)
Comment (aasampaio3006) [reply examples/cdi-basic/README_pt.adoc]: Daniel,
I have to use double quotes "",  in did of using simple ''?

small change :   
   -  change  "Add README.pt.adoc Translate to portuguese: examples cdi basic TOMEE " to "Translate to portuguese: examples cdi basic TOMEE-2516"
    - insert small description with link of JIRA

I did not understand! 

Would you help me, please.

Comment (Daniel-Dos) [reply examples/cdi-basic/README_pt.adoc]: word "arquivo" two times  :  

change : 

Para usar `@Inject`, a primeira coisa que você precisa de um arquivo `META-INF/beans.xml` arquivo no modulo

to 

Para usar `@Inject`, a primeira coisa que você precisa de um arquivo `META-INF/beans.xml`  no modulo

Comment (Daniel-Dos) [reply examples/cdi-basic/README_pt.adoc]: change "Add README.pt.adoc Translate to portuguese: examples cdi basic TOMEE " to "Translate to portuguese: examples cdi basic TOMEE-2516"

change title of your PR to  => Translate to portuguese: examples cdi basic TOMEE-2516

and add the link of your ticket JIRA

Comment (aasampaio3006) [reply examples/cdi-basic/README_pt.adoc]: Ok ,
I will strive to improve next time.
Thank you very much

Comment (Daniel-Dos) [reply examples/cdi-basic/README_pt.adoc]: great, thanks my friend :  )

**Comment by dblevins** [comment]
@aasampaio3006 thank you so much for the PR!!!
@Daniel-Dos thank you for the careful review!  Mark it approved when ready and I'll merge it :)

**Comment by aasampaio3006** [comment]
Hi, @Daniel-Dos 

Sorry for the delay,
Today I finished the error.
I could check Daniel, if everything is correct, please.

Thanks.

**Comment by monsonhaefel** [comment]
Hi Andrade,

Thank you for your contribution in May of this year to the TomEE project.  I have a follow up question for you about your commit. Can you please email me at monsonhaefel@gmail.com?  I also work on the project.

**Pull request merged**